src/views/index.pug

```diff
@@ -10,6 +10,7 @@ html(lang="en")
     meta(name="author" content="University of Waterloo Software Engineering Class of 2021 <https://github.com/sexxis/website/contributors>")
 
     link(href="static/style.css" rel="stylesheet")
+    link(href="bootstrap/css/bootstrap.min.css" rel="stylesheet")
```

Comment:
@justinpezzack so basically `node_modules/bootstrap/dist/` will be exposed as `/bootstrap`